frontend/components/Calendar/EventCard.tsx

```diff
@@ -267,7 +266,7 @@ export default function EventCard({
         {event.organizer}
       </Typography>
 
-      {(hasAccess(apiContext, 'event:update') || authorIsUser(event.author, user)) && (
+      {(hasAccess(apiContext, 'event:update') || event.author?.id === user.id) && (
```

Comment:
This comment goes for all of the removed `authorIsUser` calls: Wouldn't it be better if they stayed in place? The function name should perhaps be something more appropriate now that we have custom authors.

Reply:
Events don't use the new author system yet so here the "author.id" is equal to  "author.member.id"

Reply:
Fair enough. Were all of the changes away from authorIsUser for events?

Reply:
Yes, should be